src/content/docs/kubernetes-pixie/kubernetes-integration/understand-use-data/find-use-your-kubernetes-data.mdx

```diff
@@ -2423,7 +2455,27 @@ Query the `K8sApiServerSample` [event](/docs/telemetry-data-platform/understand-
       </td>
 
       <td>
-        Number of stored objects at the time of last check, split by kind.
+        Number of stored objects at the time of last check, split by kind (alpha; deprecated in Kubernetes 1.22).
```

Comment:
Maybe it is redundant but maybe it is also a bit clearer. Shall we mention here that it is replaced by `apiserverStorageObjects` in k8s 1.23+?

Reply:
Done in 0b782eae3fd7da1856370e2efed260a4fe70f300.